demisto_sdk/commands/run_cmd/runner.py

```diff
@@ -136,6 +161,44 @@ def _export_debug_log(self, log_ids: list):
                             output_file.write(line.encode('utf-8'))
             print_color(f'Debug Log successfully exported to {self.debug_path}', LOG_COLORS.GREEN)
 
+    def _return_raw_outputs_from_log(self, log_ids: list) -> dict:
+        """
+            retrieves the raw_outputs section from the debug_log
+        Args:
+            log_ids (list): artifact ids of the log files
+
+        Returns:
+            the raw_outputs of the executed query
+        """
+        if not self.debug_path:
```

Comment:
when you have an `else` statement, there is no reason to use negative conditions (for future reference)

Reply:
What do you mean? put the `else` part first in this case?

Reply:
yes